src/scripts/settings/userSettings.js

```diff
@@ -239,7 +239,7 @@ export class UserSettings {
             return this.set('enableBackdrops', val.toString(), false);
         }
 
-        return toBoolean(this.get('enableBackdrops', false), false);
+        return toBoolean(this.get('enableBackdrops', false), true);
```

Comment:
Navigation on the TV is not smooth with backdrops due to the poor performance of the TV hardware. So I would disable it for "slow" browsers.

Import `browser` module and
```suggestion
        return toBoolean(this.get('enableBackdrops', false), !browser.slow);
```